roles/deploy-aws-infra/tasks/main.yml

```diff
@@ -0,0 +1,343 @@
+---
+# tasks file for deploy-infra
+- name: Create VPC if required
+  ec2_vpc_net:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    name: "{{ env_id }}-vpc"
+    cidr_block: 172.31.0.0/16
```

Comment:
Please make this a variable and default it to this value.

Reply:
Will do

Reply:
@oybed I see a "problem" here. If we let the user decide about this network, we will need to add more complexity to the playbook in order to figure out what are the possible subnets for that network.

As this is just happening when the user delegates the VPC and all the required components creation to the role ('aws_create_vpc: true'), could we just use the values proposed for the AWS Reference Architecture[1]?

[1 ]https://github.com/openshift/openshift-ansible-contrib/blob/master/reference-architecture/aws-ansible/playbooks/vars/main.yaml#L10-L11

Reply:
@makentenza ok - let's do a follow-up PR to "fix it" later.